### DISCUSSION TITLE
plug install failed online

### DISCUSSION THREAD

**Discussion by MagicVin** [arcticicestudio/nord#228] [Help & Support]
Is there a way to install nord-vim offline?
I can't install plug online since the network police in China.

Will
Thanks!

**Comment by felix-kyun** [comment]
On the installation page you will find the manual section.
Link:https://www.nordtheme.com/docs/ports/vim/installation

```
Manual
To install Nord Vim manually, download the latest version or clone the repository.
Afterwards copy the nord.vim file into the .vim/colors located in your home directory.
```

**Comment by svengreb** [comment]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.
